src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from 'react-router';
+import Home from './components/Home';
+import './App.css';
+import PlayerSearch from './components/PlayerSearch';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <div>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/player-search/:season" element={<PlayerSearch />} />
+      </Routes>
+    </div>
```

Comment:
- [ ] Dear @RicardoEgas, Cloneing the project locally, I can see your app lacks some major features of the app. Please consider adding the following features which are listed in the project requirements. 
- 1. The Home page has a list of categories that could be filtered using given parameters, e.g. category name.
- 2. There's a link from the `Detail` page to the `Home` page by clicking the "Back" button.
- 3. You should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) for `Typography` and `Layout (spacing and sizing)`. Currently, your design needs an adjustment. 
> To summarize, Your homepage displays a list of numbers, Which does not communicate anything on its own. and the layout is not arranged as shown in the original design. Please follow the design closely and make sure the expected functionalities are in place.